packages/lpa-questionnaire-web-app/src/controllers/development-plan.js

```diff
@@ -0,0 +1,80 @@
+const { VIEW } = require('../lib/views');
+const getAppealSideBarDetails = require('../lib/appeal-sidebar-details');
+const { getTaskStatus } = require('../services/task.service');
+const { createOrUpdateAppealReply } = require('../lib/appeal-reply-api-wrapper');
+
+const sectionName = 'optionalDocumentsSection';
+const taskName = 'developmentOrNeighbourhood';
+
+exports.getDevelopmentPlan = (req, res) => {
+  const developmentPlanSection =
+    req.session.appealReply.optionalDocumentsSection.developmentOrNeighbourhood;
```

Comment:
What happens if this bugs out? (ie part of the object doesn't exist) - does it get handled by the main error handler?

Reply:
Perhaps worth some optional chaining for this?

Reply:
Is this part of this ticket a a larger change for all for a separate ticket?